lib/matplotlib/axes/_base.py

```diff
@@ -2436,14 +2436,19 @@ def autoscale_view(self, tight=None, scalex=True, scaley=True):
         if tight is not None:
             self._tight = bool(tight)
 
-        if self.use_sticky_edges and (
-                (self._xmargin and scalex and self._autoscaleXon) or
-                (self._ymargin and scaley and self._autoscaleYon)):
-            stickies = [artist.sticky_edges for artist in self.get_children()]
-        else:  # Small optimization.
-            stickies = []
-        x_stickies = np.sort([x for sticky in stickies for x in sticky.x])
-        y_stickies = np.sort([y for sticky in stickies for y in sticky.y])
+        x_stickies = y_stickies = np.array([])
```

Comment:
Doesn't this make x_stickes and y_stickies the same thing?

Reply:
You can still assign to them separately.  Just not *modify* them in-place separately.